platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -54,6 +55,10 @@
 
 	private final GodotView godotView;
 	private final InputManagerCompat inputManager;
+	private final static int MOUSE_WHEEL_UP = 1 << 3; // uses global input indexes
+	private final static int MOUSE_WHEEL_DOWN = 1 << 4;
+	private final static int MOUSE_WHEEL_RIGHT = 1 << 6;
+	private final static int MOUSE_WHEEL_LEFT = 1 << 5;
```

Comment:
These are no longer needed here.

Reply:
I guess these directly reflect `ANDROID_MOUSE_WHEEL_UP` etc. from the c++?

This needs a better mechanism to make sure it doesn't break in the future if a constant changes somewhere.